src/scripts/showNsfw.js

```diff
@@ -0,0 +1,26 @@
+import { getOptions } from './utils/jsTools.js';
+
+let intervalId;
+
+function showPosts(handles) {
+  for (let handle of handles) {
+    if ($('header .aspect-square + span').text() === '@' + handle) {
```

Comment:
a better way to check the active project is to import it from `user.js`. the check could also be done just once in the `main` function, because you can't switch projects without reloading the page, so it shouldn't need to check every single time.